concepts/basics/introduction.md

```diff
@@ -1,18 +1,18 @@
 # Introduction
 
-Java is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable is defined by explicitly specifying its type.
+Java is a statically-typed language, which means that the type of a variable is known at compile-time. Assigning a value to a name is referred to as defining a variable. A variable is defined by explicitly specifying its type.
 
 ```java
 int explicitVar = 10;
 ```
 
-Updating a variable's value is done through the `=` operator. Once defined, a variable's type can never change.
+Updating a variable's value is done through the `=` operator. Here, `=` does not represent mathematical equality. It simply assigns a value to a variable. Once defined, a variable's type can never change.
```

Comment:
I think it might more friendly if we just explain what `=` does (not sure if the "mathematical equality" will be understood by everyone).  

```suggestion
The `=` operator assigns a value to a variable. It can also be used to update a variable's value.
```